woo-viet.php

```diff
@@ -224,4 +227,18 @@ public function add_gateway_class( $methods ) {
 
 	}
 
+	/**
+	* Enqueue scripts
+	*
+	* @author 	Longkt
+	* @since 	1.4
+	*/
+	public function load_scripts() {
+		// Enqueue style
+		wp_enqueue_style( 'wooviet-style', WOO_VIET_URL . 'assets/style.css' );
```

Comment:
If we move this to the Provinces feature, let's use: 

* `woo-viet-provinces-style` instead of `woo-viet-style`
* Filename: `assets/provinces.css` instead of `assets/style.css` 

The same should apply for the JS file below.